### PR TITLE
solved bug,issue #75

### PR DESCRIPTION
**Pull request by ludov04**

### CODE REVIEW

File: views/package_version/show.ejs [hunk 1]
```diff
@@ -18,11 +18,9 @@
         </select>
       </div>
         <% if(inViewerPane) { %>
+        <section data-package-name="<%=pkgVersion.package_name%>" data-type-id="<%=pkgVersion.package['type_id']%>" class="packageData" itemscope itemtype="http://schema.org/SoftwareApplication">
```
Comment (ludov04) [inline views/package_version/show.ejs]: you can actually put the data attributes in the top parent section, that's why I put the 
```
itemscope itemtype="http://schema.org/SoftwareApplication"
```
I should have been more clear

File: assets/js/dependencies/async_loader.js [hunk 2]
```diff
@@ -164,8 +158,7 @@
       window.packageVersionControl=function(){
         var versions = $('#packageVersionSelect').find('option');
         if(versions.length>0){
-          var packageInfo=$('.versionCheck> p:eq(0)').text().trim();
-          var packageName=packageInfo.split(',')[0];
+          var packageName = $(".packageData").data("packageName");
```
Comment (ludov04) [inline assets/js/dependencies/async_loader.js]: Isn't it `package-name` ?

Comment (HannesBuseyne) [inline assets/js/dependencies/async_loader.js]: should be solved in new commit

File: views/package_version/show.ejs [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <%- partial ('../shared/_navbar.ejs') %>
 <% var pkgVersion = data; %>
 <section class="package" itemscope itemtype="http://schema.org/SoftwareApplication">
+<section data-package-name="<%=pkgVersion.package_name%>" data-type-id="<%=pkgVersion.package['type_id']%>" class="packageData" itemscope itemtype="http://schema.org/SoftwareApplication"></section>
```
Comment (ludov04) [inline views/package_version/show.ejs]: What I meant is put the `data-*` in the section on line 3, no need to create a new empty section, you can add your class and the other data-* to the existing one ;)

File: views/topic/show.ejs [hunk 4]
```diff
@@ -18,6 +18,8 @@
       <p><%- topic.description %></p>
     </section>
 
+    <section data-package-name="<%=topic.package_version.package_name%>" class="packageData" itemscope itemtype="http://schema.org/SoftwareApplication">
```
Comment (ludov04) [inline views/topic/show.ejs]: Same here, place the `data-*` and add your class to the root section

File: views/package_version/show.ejs (hunk 3 above)
Comment (HannesBuseyne) [inline views/package_version/show.ejs]: should be solved :)

**Pull request merged**